app/controllers/jobseekers/sessions_controller.rb

```diff
@@ -1,3 +1,4 @@
 class Jobseekers::SessionsController < Devise::SessionsController
   after_action :replace_devise_notice_flash_with_success!, only: %i[create destroy]
+  before_action :sign_out_publisher!, only: %i[create]
```

Comment:
So what do these bangs denote?

Reply:
Destructive, I suppose. It changes stuff in-place. Bit of a grey area though. Happy to have it without if that's nicer?

Reply:
I don't mind, just wanted to make sure we have a common understanding of what they mean